README.md

```diff
@@ -1,2 +1,22 @@
-# snap-opensearch
-OpenSearch Snap
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
+# OpenSearch-Snap
+This is the snap for OpenSearch. It works on Ubuntu, Fedora, Debian, and other major Linux :penguin: distributions. This snap creates a single-node cluster that is **not production ready**. Consider using the [opensearch-operator](https://github.com/canonical/opensearch-operator) to have an out of the box :gift: experience.
+
+## Install
+```shell
+sudo snap install opensearch
+```
+[Don't have snapd installed?](https://snapcraft.io/docs/core/install)
+
+## Configuration
+The cluster can be configured by editing `jvm.options` and `opensearch.yml` at `/var/snap/opensearch/common/config`. Check the [OpenSearch documentation](https://opensearch.org/docs/latest) for more details.
```

Comment:
This section should also explain what the `install` hook does. It is expected that if no configurations exist in `/var/snap/opensearch/common`, the service will start with a default config in place (and the node named `node-1`). Are there any implications around this? Example: the charm installs the snap, then tries to update the config files (e.g. changing the node name and other options), and then restarts the service. Could data be corrupted (e.g. indices created by the previous config) in these cases?

Reply:
I've added a better explanation about the `install` hook.
Changing a cluster or node name doesn't affect existent indices. They well keep it and will be accessible :slightly_smiling_face: